packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -447,16 +447,17 @@ export default class ReconnectionManager {
   }
 
   /**
+   * @param {Object} options passes mediaSetting and streams if needed
    * @returns {Promise}
    * @private
    * @memberof ReconnectionManager
    */
-  reconnectMedia() {
+  reconnectMedia(options = {}) {
     LoggerProxy.logger.log('ReconnectionManager:index#reconnectMedia --> Begin reestablishment of media');
 
     ReconnectionManager.setupPeerConnection(this.meeting);
 
-    return Media.attachMedia(this.meeting.mediaProperties, {
+    return Media.attachMedia({...this.meeting.mediaProperties, ...options}, {
```

Comment:
is this really needed? I see you're already updating the `meeting.mediaProperties.mediaDirection` before calling reconnectMedia() in moveTo. I'd rather avoid this change here, because it makes it all a bit messy - someone can call  reconnectMedia with options that don't match `meeting.mediaProperties.mediaDirection` and then you will end up with media flowind in directions that don't match what is in `meeting.mediaProperties.mediaDirection`

Reply:
yep is not needed , previously i was sending streams as part of it